plinth/modules/tor/templates/tor.html

```diff
@@ -53,7 +53,11 @@
       {% endif %}
     </p>
 
-    {% include "diagnostics_button.html" with module="tor" %}
+    {% if status.is_running %}
+      {% include "diagnostics_button.html" with module="tor" %}
```

Comment:
Hi again, I've redone it in the way you described, I wasn't aware of how easy it is to pass variables like that. I like this solution much better, it's way less cruft, and sets a much better example if we manipulated buttons in the future.
Thanks, I'm learning a lot!

I was also wrestling with git a bit, sorry for the mess (had to undo an unfortunate weblate merge in between). Make sure you squash everything down. (Could I have helped there, apart from creating another PR?)